src/param.example.js

```diff
@@ -12,24 +12,32 @@ param.grey_radius = 25; // radius of grey dot
 param.display_time = 1.5; // time in seconds to display encoding trial
 param.fixation_time = 0.5; // time in seconds to display fixation trial
 param.break_duration = 60; // time in seconds to have a break between blocks
-param.encodingBlocks = 1; // number of encoding blocks
-param.repPerBlock = 1; // number of repetitions per exemplar encoding block
+param.encodingBlocks = 6; // number of encoding blocks
+param.repPerBlock = 6; // number of repetitions per exemplar encoding block
 param.trialsPerEncodingBlock =
   param.exemplarTypes.reduce((acc, t) => acc + t.length, 0) *
   param.repPerBlock *
   param.exemplarTypes.length; // total number of trials per block
 param.foilTestedOn = [
-  1,
   0,
+  2,
   0,
-  1
+  2,
+  2,
+  2,
+  0,
+  0
 ]; /* for every exemplar, the index of the image which the foil is
       different from the original triplet */
 param.foilTestedType = [
   false,
   true,
   false,
-  true
+  true,
```

Comment:
We might want to specify the repeating behavior in a comment, but this looks good for now

Reply:
cool, thanks